charts/tfy-notebook-controller/templates/oauth-secret.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.oauth.enabled -}}
```

Comment:
What happens if the `.Values.oauth` field is absent? Usually, do we need to handle that edge case?